src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -194,6 +196,45 @@ public static bool IsReferenceOrContainsReferences<T>()
             // See getILIntrinsicImplementation for how this happens.
             throw new InvalidOperationException();
         }
+
+        // Returns true iff the object has a component size;
+        // i.e., is variable length like string, array, Utf8String.
```

Comment:
Nit: We do not have `Utf8String` and it is unclear whether we will ever have it. It may be a bit premature to start mentioning it in comments.

Reply:
Good eye - this was an oversight on my part. This code was copied from the feature/utf8string branch and I forgot to clean up the comments. Will fix.

Reply:
Utf8String still mentioned